README.md

```diff
@@ -4,15 +4,22 @@ Exposing DanielSWolf / rhubarb-lip-sync as a service
 
 ## Docker setup
 
```

Comment:
@mermop this docker works for me, would you do the honors of running it on your machine, or, if you don't want to be burdened with Docker we could merge this and I could start another branch to get this working on Heroku?